data/kimse2020Viral/kimse2020code.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
+
+
+import yaml
+import pandas as pd
+import numpy as np
+class folded_str(str): pass
+class literal_str(str): pass
+
+def folded_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='>')
+def literal_str_representer(dumper, data):
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer)
+
+
+# In[2]:
+
+
+data = pd.read_csv("CombinedDataset-2.csv") 
+data=data[113:132]
```

Comment:
This *should* work provided that the underlying dataset doesn't change. But maybe we can be more explicit using a filter? Something like `data = data[data.some_study_identifier == target_study_identifier]`?